aesara/link/numba/dispatch/basic.py

```diff
@@ -636,43 +636,48 @@ def advancedincsubtensor1(x, vals, idxs):
         return advancedincsubtensor1
 
 
+@numba_njit(inline="always")
+def deepcopyop_1(x):
+    return x
+
+
+@numba_njit(inline="always")
```

Comment:
Reminder: we should revisit our liberal use of `inline="always"`; it's likely causing us to spend more time compiling things in Numba.